addons/website_livechat/models/discuss_channel.py

```diff
@@ -25,26 +25,6 @@ def channel_pin(self, pinned=False):
             self.sudo().unlink()
 
     def _field_store_repr(self, field_name):
-        if field_name == "visitor":
-            return [
-                Store.Attr(
-                    "visitor",
-                    lambda channel: Store.One(
-                        channel.livechat_visitor_id,
-                        [
-                            "country",
-                            "history",
-                            "is_connected",
-                            "lang_name",
-                            "name",
-                            "partner_id",
-                            "website_name",
-                        ],
-                    ),
-                    predicate=lambda channel: channel.livechat_visitor_id
```

Comment:
We might want to keep the predicate (we actually need the access check part to avoid ACL issues).